apps/ex_wire/lib/ex_wire/packet/status.ex

```diff
@@ -63,7 +65,7 @@ defmodule ExWire.Packet.Status do
       protocol_version: ExWire.Config.protocol_version(),
       network_id: ExWire.Config.network_id(),
       total_difficulty: packet.total_difficulty,
-      best_hash: packet.genesis_hash,
```

Comment:
`genesis_hash` was returned here, previously, I think so say: "Our best hash is the genesis hash, because we know nothing!" We may want to (when there's no sync) just look at the current chain and return its `genesis_hash` as our best hash, etc, to correctly state we know nothing.

Reply:
@hayesgm what is meant by _current chain_? Something like `genesis_hash = Chain.load_chain(@network_id_to_chain_name[packet.network_id]).genesis.parent_hash`?

Reply:
Hm, should it just be `genesis.hash` instead of `genesis.parent_hash`? I honestly don't know which is correct.

Reply:
I'm not sure either. If `Genesis` had a `block_hash` ([I don't think it does](https://github.com/mana-ethereum/mana/blob/master/apps/blockchain/lib/blockchain/genesis.ex#L17)), I think it'd be correct. Could use `Genesis.create_block(Chain.t(), TrieStorage.t())` with our trie or an empty one. I can update to that if you think it makes sense.